inductiva/utils/format_utils.py

```diff
@@ -50,20 +80,21 @@ def apply_formatters(table_data: dict, formatters: dict):
             apply to that column's data.
     """
     output_table_data = copy.deepcopy(table_data)
-    for column_name, formatter in formatters.items():
+    for column_name, forms in formatters.items():
```

Comment:
What is a forms and why is it different than formatters? :)

Reply:
`forms` is a list of formatters. I just cannot call it that because it is redefined the scope of `formatters` in the function arguments

Reply:
`formatters_list`?

Reply:
`formatters_for_column_with_name_columnName`?
😉